donut.c

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+#include <math.h>
+#include <stdint.h>
+#include <string.h>
+#include <unistd.h>
+
+int main() {
+    float A = 0, B = 0;
+    float i, j;
+    int k;
+    float z[1760];
+    char b[1760];
+    printf("\x1b[2J");
+    for(;;) {
```

Comment:
Adding a comment explaining the purpose of the `memset` calls would improve clarity.